Python/config.py

```diff
@@ -45,11 +54,18 @@
 
 		},
 
+		"publicVariables" : {
+
+			"IncludeDir" : "{buildDir}/lib/Python.framework/Headers",
+			"LibDir" : "{buildDir}/lib/Python.framework/Versions/{pythonVersion}/lib",
```

Comment:
Should these be `pythonIncludeDir` and `pythonLibDir`?

Reply:
Indeed! Thanks for spotting that - fixed in f303ef6.